readme.md

```diff
@@ -13,25 +13,31 @@ npm install browserstack
 ## Usage
 
 ```javascript
-var BrowserStack = require( "browserstack" );
+var BrowserStack = require( "browserstack" ),
+	BrowserStackCredentials = {
+		username: "foo",
+		password: "p455w0rd!!1"
+	};
 
 // REST API
-var client = BrowserStack.createClient({
-	username: "foo",
-	password: "p455w0rd!!1"
-});
+var client = BrowserStack.createClient(BrowserStackCredentials);
 
 client.getBrowsers(function( error, browsers ) {
 	console.log( "The following browsers are available for testing" );
 	console.log( browsers );
 });
 
-// Screenshots API
-var screenshotClient = BrowserStack.createScreenshotClient({
-	username: "foo",
-	password: "p455w0rd!!1"
+// Automate API
+var automateClient = BrowserStack.createAutomateClient(BrowserStackCredentials);
+
+automateClient.getBrowsers(function( error, browsers ) {
```

Comment:
The README is still very spacey since it's not checked by JSCS. The spaces around the parameters shouldn't exist, but the existing code blocks in the README also need to be updated.